src/config.js

```diff
@@ -0,0 +1,12 @@
+export const MEETING_ROOM_AVAILABILITY = '/rest/meetingRoom/availability/FIFTY_ONE';
```

Comment:
we just have '/rest/meetingRoom' replicated in a lot of places, just to clean things up a little. Doesn't matter that much though :-P